attribute.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+# Represents a classroom.
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
+    student.classroom = self
+  end
+end
+
+# Represents a Student.
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+end
```

Comment:
I'm sorry that these issues weren't pointed out earlier by the previous reviewer, I use the \[OPTIONAL\] flag though but are `highly` recommended 🙏

[OPTIONAL] - According to the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md):

As you have created the has-many side (a classroom has many students), kindly add a method that serves as the `belongs-to` side (a student belongs to a classroom).

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

So that any time we add a student to a classroom the classroom for the student is set as well, and vice versa. (i.e both ways relationships) 🤗

Link to the [example in a previous lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)